src/Libraries/RevitNodes/Elements/Material.cs

```diff
@@ -101,6 +101,186 @@ public static Material ByName(string name)
 
         #endregion
 
+        #region Properties
+
+        /// <summary>
+        /// Get Material Name
+        /// </summary>
+        /// <returns></returns>
+        public string Name
+        {
+            get { return this.InternalMaterial.Name; }
+        }
+
+        /// <summary>
+        /// Get Shininess
+        /// </summary>
+        public int Shininess
+        {
+            get { return this.InternalMaterial.Shininess; }
+        }
+
+        /// <summary>
+        /// Get Smoothness
+        /// </summary>
+        public int Smoothness
+        {
+            get { return this.InternalMaterial.Smoothness; }
+        }
+
+        /// <summary>
+        /// Get Transparency
+        /// </summary>
+        public int Transparency
+        {
+            get { return this.InternalMaterial.Transparency; }
+        }
+
+        /// <summary>
+        /// Get SurfacePatternColor
+        /// </summary>
+        public DSCore.Color SurfacePatternColor
+        {
+            get { return ToDSColor(this.InternalMaterial.SurfacePatternColor); }
+        }
+
+        /// <summary>
+        /// Get Material Class
+        /// </summary>
+        public string MaterialClass
+        {
+            get { return this.InternalMaterial.MaterialClass; }
+        }
+
+        /// <summary>
+        /// Get Material category
+        /// </summary>
+        public string MaterialCategory
+        {
+            get { return this.InternalMaterial.MaterialCategory; }
+        }
+
+        /// <summary>
+        /// Get cut pattern color
+        /// </summary>
+        public DSCore.Color CutPatternColor
+        {
+            get { return ToDSColor(this.InternalMaterial.CutPatternColor); }
+        }
+
+        /// <summary>
+        /// Get color
+        /// </summary>
+        public DSCore.Color Color
+        {
+            get { return ToDSColor(this.InternalMaterial.Color); }
+        }
+
+        /// <summary>
+        /// Get cut pattern id
+        /// </summary>
+        public Autodesk.Revit.DB.ElementId CutPatternId
```

Comment:
I don't think we want to expose the elementID here, don't we have some way to wrap this?